src/views/accounts/AccountPush/AccountPushExecutionList.vue

```diff
@@ -1,163 +1,30 @@
-            }
-          },
-          node_amount: {
-            label: this.$t('NodeOfNumber'),
-            formatter: function(row) {
-              return <span>{row.snapshot.node_amount}</span>
-            }
-          },
-          status: {
-            label: this.$t('Result')
-          },
-          actions: {
-            formatterArgs: {
-              hasDelete: false,
-              hasUpdate: false,
-              hasClone: false,
-              extraActions: [
-                {
-                  name: 'log',
-                  type: 'primary',
-                  can: this.$hasPerm('accounts.view_pushaccountexecution'),
-                  title: this.$t('Log'),
-                  callback: function({ row }) {
-                    openTaskPage(row['id'])
-                  }
-                },
-                {
-                  name: 'report',
-                  title: this.$t('Report'),
-                  can: this.$hasPerm('accounts.view_pushaccountexecution'),
-                  callback: function({ row }) {
-                    vm.visible = true
-                    vm.reportUrl = `/api/v1/accounts/push-account-executions/${row.id}/report/`
-                  }
-                },
-                {
-                  name: 'record',
-                  title: this.$t('Record'),
-                  can: this.$hasPerm('accounts.view_pushsecretrecord'),
-                  callback: function({ row }) {
-                    return this.$router.push({
-                      name: 'AccountPushList',
-                      query: {
-                        tab: 'AccountPushRecord',
-                        execution_id: row.id
-                      }
-                    })
-                  }
-                }
-              ]
-            }
-          }
-        }
-      },
-      headerActions: {
-        searchConfig: {
-          getUrlQuery: true,
-          options: [
-            {
-              label: this.$t('TaskID'),
-              value: 'automation_id'
-            },
-            {
-              label: this.$t('DisplayName'),
-              value: 'automation__name'
-            }
-          ]
-        },
-        hasSearch: true,
-        hasRefresh: true,
-        hasRightActions: true,
-        hasLeftActions: false,
-        hasMoreActions: false,
-        hasExport: false,
-        hasImport: false
-      }
-    }
-  },
-  mounted() {
-    const automation_id = this.$route.query.automation_id
-    if (automation_id !== undefined) {
-      this.tableConfig.url = `${this.tableConfig.url}?automation_id=${automation_id}`
+      url: '/api/v1/accounts/push-account-executions/',
+      automationRoute: 'AccountPushDetail',
+      detailRoute: 'AccountPushExecutionDetail',
+      resource: 'pushaccountexecution',
+      resourceLabel: this.$t('AccountPushExecution'),
+      detailDrawer: () => import('@/views/accounts/AccountPush/ExecutionDetail/index.vue')
     }
   }
 }
```

Comment:
The provided code is largely consistent with no notable irregularities or issues found. In terms of optimizations:

- Code could benefit from removing unnecessary imports at the top, such as `import { openTaskPage } from '@/utils/jms'`, though these seem to be for a separate utility component that isn't used directly in this specific template.

The current structure seems to follow the project's logic and conventions well without obvious flaws or improvements. For more detailed assistance, please specify areas you would like me to focus on.